CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set (CMAKE_TRY_COMPILE_CONFIGURATION ${CMAKE_BUILD_TYPE})
 project (openturns)
 
 option (USE_BISON                    "Looks for Bison if true and then build parser"                         ON)
+option (USE_CUBA                     "Use Cuba library"                                                      ON)
```

Comment:
entries should be added to PlatformInfo.cxx, and doc/developer/architecture.rst, in the third-party depencies section